UnofficialCrusaderPatch/Version.cs

```diff
@@ -624,6 +624,39 @@ public class Version
 
             #region AI LORDS
 
+            new Change("ai_resources_rebuy", ChangeType.AILords, true)
+            {
+                // vanilla:
+                // additional attack troops = factor * attack number
+
+                new SliderHeader("ai_nowood_maxtime", true, 0, 72, 1, 36, 1)
```

Comment:
Please document what this feature does. What does the original value of 36 (0x24) mean and why should its suggested value be 1?

This feature is on by default and set to a very much lower number than the original. This needs a good testing before we release it, it should not cripple any AI.